packages/SwingSet/src/liveslots/liveslots.js

```diff
@@ -446,14 +484,15 @@ function build(
   // use a slot from the corresponding allocateX
 
   function allocateExportID() {
-    const exportID = nextExportID;
-    nextExportID += 1;
-    return exportID;
+    return allocateNextID('exportID', 1);
```

Comment:
We didn't have one before, but let's add a note that we use "1" because it's bigger than the "o+0" that's hard-coded for the root object.

The "5" for promises is arbitrary and could be reduced to 0 without problems, but the inital `exportID` must never be less than 1.